flashflood/__init__.py

```diff
@@ -69,15 +69,30 @@ def put(self, data, event_id: str=None, date: datetime=None):
         self._upload_journal(_Journal(journal_id, manifest, io.BytesIO(data)), is_new=True)
         return Event(event_id, date, data)
 
-    def journal(self, number_of_events=10):
+    def journal(self, minimum_number_of_events=100, minimum_size=None):
+        assert 2 <= minimum_number_of_events
```

Comment:
Why was the hard minimum set to `3`?  (I maybe would expect `1`?)

Reply:
Using `1` would result in a null-op